agenda/presentation/src/main/java/com/gkp/agenda/presentation/util/Util.kt

```diff
@@ -0,0 +1,46 @@
+package com.gkp.agenda.presentation.util
+
+import java.time.LocalDate
+private const val CURRENT_AND_NEXT_SIX_DATES = 7
+
+fun getFullNameInitials(fullName: String): String {
+    val names = fullName.split(" ")
+    val initialsBuilder = StringBuilder()
+    if (names.size == 1) {
+        initialsBuilder.append(names[0].take(2).uppercase())
+        return initialsBuilder.toString()
+    }
+    if (names.size == 2) {
+        initialsBuilder.append("${names[0].first()}${names[1].first()}".uppercase())
+        return initialsBuilder.toString()
+    }
+    if (names.size >= 3) {
+        initialsBuilder.append("${names[0].first()}${names[2].first()}".uppercase())
+        return initialsBuilder.toString()
+    }
+    return ""
+}
```

Comment:
This function in general deals well with the happy path. In reality, the happy path is not always met however. A few considerations:
1. What about a name like `Philipp  Lackner` (with two spaces in between)? This would crash here, since it would result in a splitted list `["Philipp", "", "Lackner"]` where you then might try to access the first character of an empty string.
2. What about weird/invalid inputs? What if a user makes their name full of emojis? That's not explicitly in the requirements, but generally those would be valid inputs, so we as devs should also think about a nice and reasonable way to deal with that.

Reply:
To overcome "1". I filtered the list again empty  and blank string